libraries/aws_iam_role.rb

```diff
@@ -36,6 +37,29 @@ def initialize(opts = {})
         @permissions_boundary_arn  = resp.permissions_boundary.permissions_boundary_arn
       end
     end
+    fetch_role_policies(opts[:role_name])
+    fetch_attached_role_policies(opts[:role_name])
+  end
+
+  def fetch_role_policies(role_name = '')
+    catch_aws_errors do
+      resp = @aws.iam_client.list_role_policies({ role_name: role_name })
+      @inline_policies = resp.policy_names
+    end
+  end
+
+  def fetch_attached_role_policies(role_name = '')
```

Comment:
`validate_parameters` will fail if `role_name` is `''` so this default can be removed